LoopKit/InsulinKit/HKQuantitySample+InsulinKit.swift

```diff
@@ -57,6 +60,9 @@ extension HKQuantitySample {
             if dose.type == .tempBasal {
                 metadata[MetadataKeyProgrammedTempBasalRate] = HKQuantity(unit: .internationalUnitsPerHour, doubleValue: dose.unitsPerHour)
             }
+            if dose.type == .suspend {
+                metadata[MetadataKeyDuringSuspend] = true
+            }
```

Comment:
Should this be:

```suggestion
            metadata[MetadataKeyDuringSuspend] = dose.type == .suspend
```

if not, why not? Could `duringSuspend` below just be non-optional, like `manuallyEntered` is?  That would avoid awkward dealings with the optional bool.

Reply:
No longer necessary. Removing `duringSuspend` per other PR feedback.

Reply:
Actually, necessary here, but not in DoseEntry. Made non-optional per feedback.